### PR TITLE
prod hacks

### PR DESCRIPTION
**Pull request by vilmibm**
these are the hacks i made in prod to make things work

### CODE REVIEW

File: arkestrator/wsgi.py [hunk 1]
```diff
@@ -1,3 +1,9 @@
-import django.core.handlers.wsgi
+import django.core.wsgi
+import sys
 
-application = django.core.handlers.wsgi.WSGIHandler()
+sys.path.append("/opt/arkestrator/virtualenv/src/arkestrator")
```
Comment (adreyer) [inline arkestrator/wsgi.py:4]: Can we at least use `__file__` here?

Comment (croddy) [reply arkestrator/wsgi.py:4]: I agree, but want to point out we also  have the same path in `settings.py` so it should be changed in both places.

Comment (vilmibm) [reply arkestrator/wsgi.py:4]: __file__ is just the name of the running file

**Comment by vilmibm** [comment]
going to merge this in the interest of not having working tree be dirty in prod + getting bbcode fix in

**Pull request merged**